Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBService/ChangeStreamType.swift

```diff
@@ -0,0 +1,40 @@
+// swiftlint:disable weak_delegate
+import Foundation
+
+/**
+ A class representing the possible types of change streams.
+*/
+public class ChangeStreamType<T: Codable> {
```

Comment:
[nit] can you add a comment explaining what this type parameter does, or call it `DocumentT`?